x-pack/plugins/maps/public/lazy_load_bundle/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AnyAction } from 'redux';
+import { Embeddable, IContainer } from '../../../../../src/plugins/embeddable/public';
+import { LayerDescriptor } from '../../common/descriptor_types';
+import { MapStore, MapStoreState } from '../reducers/store';
+import { EventHandlers } from '../reducers/non_serializable_instances';
+import { RenderToolTipContent } from '../classes/tooltips/tooltip_property';
+import { MapEmbeddableConfig, MapEmbeddableInput, MapEmbeddableOutput } from '../embeddable/types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { IndexPatternsService } from '../../../../../src/plugins/data/common/index_patterns/index_patterns/index_patterns';
```

Comment:
Same comment as above, we can use `src/plugins/data/public/index_patterns/index.ts`